.pre-commit-config.yaml

```diff
@@ -56,14 +56,14 @@ repos:
         args: ["--fix", "--show-fixes"]
 
   # 2023-12-11: Not use mypy for now.
-  # - repo: https://github.com/pre-commit/mirrors-mypy
-  #   rev: "v1.8.0"
-  #   hooks:
-  #     - id: mypy
-  #       files: src|tests
-  #       args: []
-  #       additional_dependencies:
-  #         - pytest
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: "v1.8.0"
+    hooks:
+      - id: mypy
+        files: src
+        args: ["--ignore-missing-imports"]
```

Comment:
Which imports need to be ignored?

Reply:
Mypy for some reason get's confused with imports. Since this is already being checked by ruff, we can ignore for mypy.